tasks/agent.py

```diff
@@ -855,7 +855,9 @@ def clean(ctx):
 
 
 @task
-def version(ctx, url_safe=False, omnibus_format=False, git_sha_length=7, major_version='7', version_cached=False):
+def version(
+    ctx, url_safe=False, omnibus_format=False, git_sha_length=7, major_version='7', version_cached=False, pipeline_id=""
```

Comment:
nit: Wouldn't `None` be more appropriate for `pipeline_id` default value?

Reply:
Yes I agree!